MatrixSDK/MXSession.m

```diff
@@ -2832,7 +2832,7 @@ - (void)fixRoomsSummariesLastMessageWithMaxServerPaginationCount:(NSUInteger)max
     
     for (MXRoomSummary *summary in self.roomsSummaries)
     {
-        if (!summary.lastMessage)
+        if (!summary.lastMessage || !summary.lastMessage.text)
```

Comment:
It supposes that the code using the SDK has this specific behavior but we cannot guarantee it.

As we already have a dispatchGroup here, we could fetch the `MXEvent` to check that it was successfully decrypted.

Reply:
You're right. Updated as you said, kit PR closed.